### ISSUE TITLE
M05_L14_Enable_Dynamic_Configuration_and_Feature_Flags: Button name

### ISSUE THREAD

**Issue by mkennten** [MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions#386]
### Contact Details

mkennten@microsoft.com

### What happened?

Exercise : 01
Task: 01
Step: 02

When you run this lab you probably have already the project by the same name created and also have some pipelines created. If that's the situation then there's no "Create Pipeline" button under Pipelines/Pipelines" but instead "New pipeline".

You probably want to cover both scenarios where it's either a new project with no pipeline or an existing one with at least one existing pipeline.

And btw: I cannot select this lab from the 'Lab' list below, it's not in there.

### Lab

Other

### Relevant screenshots

![image](https://user-images.githubusercontent.com/37703728/219050149-15f9d2da-f71b-4000-bb04-4414e0216aef.png)

### Do you want to help us? 👏

- [ ] Create a Pull Request by following this project's [Contribution Guide](https://github.com/MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions/blob/master/.github/CONTRIBUTING.md#how-can-i-collaborate)

**Comment by fimdim** [comment]
@mkennten Exercise 1 is marked as **(skip if done)**, so no need to run those steps if it was done in the previous labs.
But anyway, I will update this to include the **New pipeline** button also.

**Comment by mkennten** [comment]
I agree, but the instructions are not clear in this case. In my case I hadn't done the following:
"In this exercise, you will import and run the CI pipeline, configure the service connection with your Azure Subscription and then import and run the CD pipeline." Therefore, I did not skip this.

I didn't have the CI pipeline and did not yet have the service connection setup, but I had other pipelines. And therefore, I didn't see the "Create pipeline" button. Isn't the idea that the labs can be done in any order?

**Comment by fimdim** [comment]
Yes, I created a PR #458 where I added that.
